sqlite.cpp

```diff
@@ -157,7 +343,7 @@ bool SQLite::bind_args(sqlite3_stmt *stmt, Array args) {
 		}
 
 		if (retcode != SQLITE_OK) {
-			print_error("Query failed, an error occured while binding argument" + itos(i + 1) + " of " + itos(args.size()) + " (SQLite errcode " + itos(retcode) + ")");
+			print_error("SQLiteQuery failed, an error occured while binding argument" + itos(i + 1) + " of " + itos(args.size()) + " (SQLite errcode " + itos(retcode) + ")");
```

Comment:
Review with peers mentioned that the error message is not "SQLiteQuery" specific, so it should be just "Query".